charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -38,17 +38,19 @@ Arguments (dict):
 {{- if eq ( include "trustification.application.metrics.enabled" .root ) "true" }}
 - name: METRICS
   value: "enabled"
+- name: OTEL_METRIC_EXPORT_INTERVAL
+  value: {{ if .root.Values.metrics.otelMetricExportInterval | eq nil }}"60000"{{ else }}{{ .root.Values.metrics.otelMetricExportInterval | quote }}{{ end }}
 {{- end }}
 
 {{- if eq ( include "trustification.application.tracing.enabled" .root ) "true" }}
 - name: TRACING
   value: "enabled"
 - name: OTEL_BSP_MAX_EXPORT_BATCH_SIZE
-  value: "32"
+  value: {{ if .root.Values.tracing.otelBspMaxExportBatchSize | eq nil }}"512"{{ else }}{{ .root.Values.tracing.otelBspMaxExportBatchSize | quote }}{{ end }}
```

Comment:
Can't we just use `.root.Values.tracing.otelBspMaxExportBatchSize | default 512 | quote`?